extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -531,7 +539,7 @@ export class AmpStoryStoreService {
   /**
    * Retrieves the embed mode config, that will override the default state.
    * @return {!Object<StateProperty, *>} Partial state
-   * @private
+   * @protected
    */
   getEmbedOverrides_() {
```

Comment:
Do we allow the `_` suffix for protected props too? Do we still have the Closure Compiler behavior that marks such names as private/rename-able?

Reply:
@jridgewell @erwinmombay do either of you know?

Reply:
Yes, and yes.